src/qtgui/plotter.cpp

```diff
@@ -1525,6 +1502,7 @@ void CPlotter::draw(bool newData)
     {
         m_2DPixmap.fill(PLOTTER_BGD_COLOR);
         QPainter painter2(&m_2DPixmap);
+        painter2.translate(QPointF(0.5, 0.5));
```

Comment:
I think this change (and the corresponding change in `drawOverlay` deserves to be in its own PR, since it's unrelated to zooming.